### ISSUE TITLE
Invalid URI

### ISSUE THREAD

**Issue by camlafit** [alexblackie/gextension#1]
Hi 

I've just tried your script and get this error : 
```
./gextension search caffeine
/usr/lib/ruby/2.1.0/uri/common.rb:176:in `split': bad URI(is not URI?): https://extensions.gnome.org/extension-query/?page=1&search=["caffeine"] (URI::InvalidURIError)
        from /usr/lib/ruby/2.1.0/uri/common.rb:211:in `parse'
        from /usr/lib/ruby/2.1.0/uri/common.rb:747:in `parse'
        from /usr/lib/ruby/2.1.0/uri/common.rb:1232:in `URI'
        from ./gextension:26:in `search'
        from ./gextension:103:in `<main>'

```

**Comment by alexblackie** [comment]
Ah, interesting! I've only tested under Fedora which comes with Ruby 2.3. I just now tried it under Ruby 2.1 and can confirm it does break due to a behaviour change in how Ruby 2.1 parses CLI arguments...

I've just pushed a fix for this in commit https://github.com/alexblackie/gextension/commit/32965bcd056c57ae2e4e513c3f8c1dffd588ef59 which appears to fix the issue for me under Ruby 2.1.

Thanks for trying it out!

**Comment by camlafit** [comment]
Hi

Thanks looks ok :)
I use debian jessie, and last ruby version  is 2.1.5

```
./gextension search caffeine

Found extensions:

    Caffeine                         caffeine@patapon.info
    Caffeine for XScreenSaver        xcaffeine@asegner.gmail.com
```
